TypeCobol.Test/Parser/Programs/TypeCobol/GlobalStorageSection.rdzMix.txt

```diff
@@ -50,6 +50,7 @@ Line 32[8,48] <30, Error, Semantics> - Semantic error: TYPE 'ForbiddenTypedef' i
        01 TestCtx TYPE Bool.
        PROCEDURE DIVISION.
       *G-Date and G-TechCtx are accesible from GlobalStorage
```

Comment:
Either update the comment to signal that the variable is modified despite being marked as input or modify the test to have it marked as IN-OUT.

Reply:
Test modified by using IN-OUT
https://github.com/TypeCobolTeam/TypeCobol/tree/55e817f33a3d27d5c35869ccddd44cd00f20b192